src/documents/DataSourceAPI.md

```diff
@@ -11,12 +11,15 @@ step and other transformations during the get and set workflows.
 Concrete DataSource implementations include `fluid.dataSource.url` which read and write data over HTTP both in the
 browser and in node.js, as well as [`kettle.dataSource.file`](https://github.com/fluid-project/kettle/blob/main/docs/DataSources.md#configuration-options-accepted-by-kettledatasourcefile)
  available in node.js acting on the filesystem, and
-`fluid.prefs.cookieStore` which acts on cookies in the browser. Note that a `fluid.dataSource.url` implementation is
-only available in the browser in the upcoming Infusion 4.x. Mixin grades include [
+`fluid.prefs.cookieStore` which acts on cookies in the browser. Note that mixin grades include [
```

Comment:
It seems there is still a "note" here. Should that be moved to an additional note section below, or reworded in this paragraph?

Reply:
Now reworded